tox.ini

```diff
@@ -185,6 +185,7 @@ passenv=
     CIRCLECI
     CIRCLE_*
     DD_TRACE_AGENT_URL
+    DD_API_KEY
```

Comment:
With this change the following tests fail: https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/24213/workflows/87524a06-bc90-4eec-874a-41ebe2850f26/jobs/1645305/parallel-runs/7